hack/app-sre/kustomization.yaml

```diff
@@ -0,0 +1,39 @@
+# This kustomize configuration deploys what is required for the hive-operator
+# *without* OLM.
+# Except:
+# It does not contain a Namespace or a HiveConfig. These are created by app-sre.
+
+# The CRD resources are synced automatically by the `build-app-sre-template` make target.
+# The other resources should be curated manually.
+resources:
+- ../../config/operator/operator_role.yaml
+- ../../config/operator/operator_role_binding.yaml
+- ../../config/operator/operator_deployment.yaml
+- ../../config/crds/hiveinternal.openshift.io_clustersyncleases.yaml
```

Comment:
question: can we separate out the resources into sections that clearly show these are generated (do not touch) and these aren't

```
resources:
- curated-manually1
- curated-manually-2

## GENERATED: DO NOT TOUCH
resources:
- generated1
- generated2
```

or something of that sort?

Reply:
Sorry, no, I tried it and kustomize doesn't like it. We could probably do something similar by using separate kustomization files for those resource groups, but I _really_ dislike that kind of multi-file spread. (Bad enough the number of different files we're having to maintain as it is.)